scripts/run_tasks_for_coverage.sh

```diff
@@ -77,6 +77,9 @@ coverage run --append `which invoke` app.users.list-all
 coverage run --append `which invoke` app.users.add-role --role=Researcher --email=user@example.org
 coverage run --append `which invoke` app.users.remove-role --role=Researcher --email=user@example.org
 
+# test app.initialize.*
+coverage run --append `which invoke` app.initialize.initialize-gitlab-submissions --email user@example.org --dryrun
```

Comment:
This did not work as user@example.com was not an admin and this task checked for admin.
Check removed.